fuse_constraints/include/fuse_constraints/normal_prior_pose_3d_cost_functor.h

```diff
@@ -77,47 +77,40 @@ class NormalPriorPose3DCostFunctor
    *              (x, y, z, qx, qy, qz)
    * @param[in] b The 3D pose measurement or prior in order (x, y, z, qw, qx, qy, qz)
    */
-  NormalPriorPose3DCostFunctor(const fuse_core::Matrix6d& A, const fuse_core::Vector7d& b);
+  NormalPriorPose3DCostFunctor(const fuse_core::Matrix6d& A, const fuse_core::Vector7d& b):
+  A_(A),
+  b_(b),
+  orientation_functor_(fuse_core::Matrix3d::Identity(), b_.tail<4>())  // Delta will not be scaled
+  {
+  }
 
   /**
    * @brief Evaluate the cost function. Used by the Ceres optimization engine.
    */
   template <typename T>
-  bool operator()(const T* const position, const T* const orientation, T* residual) const;
+  bool operator()(const T* const position, const T* const orientation, T* residual) const {
```

Comment:
/tmp/ws/src/fuse/fuse_constraints/include/fuse_constraints/normal_prior_pose_3d_cost_functor.h:91:  when starting a new scope, { should be on a line by itself  [whitespace/braces] [4]